src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -254,6 +254,11 @@ module sonic-device_metadata {
                     type inet:ipv4-address;
                     description "BGP Router identifier";
                 }
+
+                leaf chassis_hostname {
```

Comment:
Wondering what is difference between hostname and chassis_hostname?

Reply:
In SONiC T2, the hostname is the Linecard or supervisor hostname. The `chassis_hostname` is the hostname of the name of the chassis to which linecard or supervisor this device belongs to.